stories/header.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Header from '../src/components/Header';
+
+const header = storiesOf('Components/Header', module)
+  .add('with title', () => 
+    <Header text="SimplyE" />
+  )
+  .add('with logout link', () => 
+    <Header text="SimplyE" logOut="./logout" />
+  );
```

Comment:
How about also adding an example of the header with a logo (`imgSrc` prop)?

Reply:
I can't get images to load so I will include that later.